index.js

```diff
@@ -31,12 +31,14 @@ try {
 
   const input = core.getInput('input');
   const output = core.getInput('output');
+  const format = core.getInput('format');
 
   console.log('Options:');
-  console.log(`  i: ${input}`);
-  console.log(`  o: ${output}`);
+  console.log(`  input:  ${input}`);
+  console.log(`  output: ${output}`);
+  console.log(`  format: ${format}`);
 
-  let command = `cyclonedx-py -r -o ${output}`
+  let command = `cyclonedx-py -r -i ${input} --format ${format} -o ${output}`
```

Comment:
this bahaviour depends heavily on specific version of `cyclonedx-py` 

will need to refactor your work, before it can be published.

Reply:
> Additionally, the GitHub Action automatically downloads the latest version of cyclonedx-py.

tries to download the latest version that pip finds passing some env-consraints.
AND ONLY IF there is no `cyclonedx-py` already installed. 

anyway, i will have the version constraint added via #7

FYI: `cyclonedx-py --version` will not be available before the upcoming `cyclonedx-py v4`